optaplanner-spring-integration/optaplanner-spring-boot-autoconfigure/src/main/java/org/optaplanner/spring/boot/autoconfigure/OptaPlannerAutoConfiguration.java

```diff
@@ -179,11 +185,14 @@ public SolverConfig solverConfig() {
             Class<?>[] entityClasses = solverConfig.getEntityClassList().toArray(new Class<?>[0]);
             ConstraintStreamImplType constraintStreamImplType =
                     solverConfig.getScoreDirectorFactoryConfig().getConstraintStreamImplType();
+            boolean droolsAlphaNetworkCompilerEnabled =
+                    solverConfig.getScoreDirectorFactoryConfig().isDroolsAlphaNetworkCompilerEnabled();
 
             return (ConstraintVerifier<ConstraintProvider_, SolutionClass_>) ConstraintVerifier.build(constraintProvider,
-                    solverConfig.getSolutionClass(),
-                    entityClasses).withConstraintStreamImplType(
-                            (constraintStreamImplType != null) ? constraintStreamImplType : ConstraintStreamImplType.DROOLS);
+                    solverConfig.getSolutionClass(), entityClasses)
+                    .withConstraintStreamImplType(
+                            (constraintStreamImplType != null) ? constraintStreamImplType : ConstraintStreamImplType.DROOLS)
+                    .withDroolsAlphaNetworkCompiler(droolsAlphaNetworkCompilerEnabled);
```

Comment:
ah, that's why the validation accepts false.
Maybe it should accept Boolean? Idunno. As is is fine too. This is serious nitpick. Feel free to ignore.